virttest/utils_misc.py

```diff
@@ -634,6 +634,23 @@ def get_full_pci_id(pci_id):
         return None
 
 
+def get_host_bridge_id(session=None):
```

Comment:
could you help move this function to utils_sys.py? We avoid adding new function into utils_misc.py.

Reply:
+1